### PR TITLE
18

### PR DESCRIPTION
**Pull request by cichan02**

### CODE REVIEW

**Comment by cichan02** [comment]
First of all merge https://github.com/cqqqt/Task-Viewer-Backend/pull/49

File: src/test/java/com/taskviewer/api/postgres/PgTasksUnitTest.java [hunk 1]
```diff
@@ -90,7 +91,6 @@ void verifiesByUsername() {
 		List<Task> tasks = new ArrayList<>(2);
 		tasks.add(task1);
 		tasks.add(task2);
-		String username = "testUsername";
 		when( jdbc.query(PgTasks.FIND_BY_USERNAME, view, username)).thenReturn(tasks);
```
Comment (h1alexbel) [inline src/test/java/com/taskviewer/api/postgres/PgTasksUnitTest.java:95]: we don't tolerate empty lines
read: https://www.yegor256.com/2014/11/03/empty-line-code-smell.html

File: src/test/resources/application-test.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+spring:
+  datasource:
+    url: ${TEST_URL}
+    username: ${TEST_USERNAME}
+    password: ${TEST_PASSWORD}
+    driver-class-name: org.testcontainers.jdbc.ContainerDatabaseDriver
+  jpa:
```
Comment (h1alexbel) [inline src/test/resources/application-test.yaml:7]: @cichan02 we are not using any ORM here

**Review by h1alexbel — CHANGES_REQUESTED** [review]
@cichan02 use 2 space indentation

**Comment by cichan02** [comment]
I see failed checks. Try to understand why failed.

**Comment by cichan02** [comment]
Я пока убрал интеграционный тест, чтобы смерджить. А потом уже разбираться.

**Review by h1alexbel — APPROVED** [review]

**Review by h1alexbel — CHANGES_REQUESTED** [review]
@cichan02 setup 2 space indentation

**Comment by cichan02** [comment]
Оно не меняется просто так!!!

**Review by h1alexbel — APPROVED** [review]

**Comment by h1alexbel** [comment]
@cichan02 fix merge conflict

**Comment by cichan02** [comment]
Cheers 🥳

**Comment by cichan02** [comment]
@h1alexbel, finally it turns in git-branches-hell. I can't solve merge conflict without hard merge. Just merge 21, which bring #18 features too.

**Review by h1alexbel — APPROVED** [review]

**Comment by h1alexbel** [comment]
@cichan02 fix the merge conflicts

**Comment by cichan02** [comment]
@h1alexbel solved all merge conflicts

**Comment by h1alexbel** [comment]
@rultor merge

**Review by h1alexbel — APPROVED** [review]

**Comment by h1alexbel** [comment]
@cichan02 thanks

**Pull request merged**